doc/deployment.md

```diff
@@ -0,0 +1,38 @@
+# Deployment of the agent service
+The project contains yaml files for Openshift deployment. This document describes the deployment process.
+By default we deploy images from `quay.io/kubev2v` namespace. We push latest images after every merge of the PRs.
+
+## Deploy on openshift
+In order to deploy the Agent service on top of Openshift there is Makefile target called `deploy-on-openshift`.
+
+```
+$ oc login --token=$TOKEN --server=$SERVER
```

Comment:
Does it matter in which namepsace to run those commands?

Reply:
It does, currently it's using the default you are in. In future we need to modify the yaml files so we can target the namespace I think.